### PR TITLE
Blog app view

### PR DESCRIPTION
**Pull request by principles31**
# Blog app - Views:
> This project aims to add the Views associated with the Models and Controllers you've already created for your Blog App.
> The following were implemented in this **PR as the project requirements:**
  - Implemented the design from the sneak peek wireframes.

  - Added basic Styling
  - Used methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

  - Added some comments and post data to the database

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
```
Comment (omarsalem7) [inline app/controllers/users_controller.rb:2]: - [x] Kindly add the views like wireframes and populate it with the data like in the [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#project-requirements)
     - add `index.html.erb` view to display user.photo, user.name and user.posts_counter.
     - Add also `app/views/users/show.html.erb` to show recent 3 posts.
     - **See all Post** links opens user's post index as displayed in wireframe
- [x] Add views for posts like wireframe also:
     - Add `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page
     - In Post show page displays post with all comments listed as shown on the wireframe.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @principles31 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
@omarsalem7 Thanks for the feedback. That will be done ASAP.

**Comment by principles31** [comment]
- [x] Kindly add the views like wireframes and populate it with the data like in the [project requirements]- (https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#project-requirements)
- add index.html.erb view to display user.photo, user.name and user.posts_counter.
- Add also app/views/users/show.html.erb to show recent 3 posts.
- See all Post links opens user's post index as displayed in wireframe
 - [x] Add views for posts like wireframe also:
- Add post.recent_comments to render 5 most recent comments in the app/views/posts/index.html.erb page
- In Post show page displays post with all comments listed as shown on the wireframe.

- [x] The requested changes above has been implemented

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @principles31,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by principles31** [comment]
@Omar-Muhamad Thank you

**Pull request merged**